### PR TITLE
Bump version.

### PR DESCRIPTION
**Pull request by liuche**
Version bump & changelog
### Pull Request checklist

- [ ] This PR includes thorough **tests** or an explanation of why it does not
- [X] This PR includes a **CHANGELOG entry** or does not need one

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -25,7 +25,7 @@ android {
         targetSdkVersion 28
         versionCode 11 // This versionCode is "frozen" for local builds. For "release" builds we
         // override this with a generated versionCode at build time.
-        versionName "3.2.2"
+        versionName "3.2.6.1"
```
Comment (liuche) [inline app/build.gradle]: I'm not sure if we should just bump per major/minor version? Or have a commit for all releases. In any case, when we do the 3.3 release, I'll update this version too, but we need the CHANGELOG changes.

Comment (Baron-Severin) [inline app/build.gradle]: Our releases tab is getting pretty hard to read.  Suggest updating this to include pre-release metadata as per https://semver.org/#spec-item-9 (e.g., `3.2.6-LAT.1`).

Comment (Baron-Severin) [reply app/build.gradle]: Not sure I'm understanding you correctly, I do think we need to tag everything that goes out to users differently though.  There will be a lot of headings under our releases tab, but I think it is important to keep that record.

**Comment by liuche** [comment]
Closing this for now, I'll make a new PR with the suggested changes.

**Pull request closed without merging**